### DISCUSSION TITLE
Download syntax

### DISCUSSION THREAD

**Discussion by Cl0udk1ck3r** [pystardust/ani-cli#362] [Q&A] [answered]
I seem not to understand the syntax to download an episode. Let's say I want to get episode 4 of .hack//SIGN to download:

ani-cli -d hacksign 4

However, when I attempt to do this, I get "no results found".
![why won't this work](https://user-images.githubusercontent.com/96272542/150243261-01a13087-f3d6-40e0-9bb5-8b5033fe865a.png)

**Comment by port19x** [comment, marked as answer]
You have to choose your episode range interactively
